src/platform/forms-system/src/js/review/ReviewPage.jsx

```diff
@@ -27,36 +31,40 @@ class ReviewPage extends React.Component {
     focusElement('h2');
   }
 
+  renderDowntime = (downtime, children) => {
+    if (downtime.status === externalServiceStatus.down) {
+      const Message = this.props.formConfig.downtime.message || DowntimeMessage;
+
+      return <Message downtime={downtime} />;
+    }
+
+    return children;
+  };
+
   render() {
-    const { formConfig, pageList, path } = this.props;
+    const { formConfig, pageList, path } = this.props.route;
 
+    const downtimeDependencies = formConfig?.downtime?.dependencies || [];
     return (
       <div>
         <ReviewChapters formConfig={formConfig} pageList={pageList} />
-        <SubmitController
-          formConfig={formConfig}
-          pageList={pageList}
-          path={path}
-        />
+        <DowntimeNotification
+          appTitle="application"
+          render={this.renderDowntime}
+          dependencies={downtimeDependencies}
+          customText={formConfig.customText}
+        >
+          <SubmitController
+            formConfig={formConfig}
+            pageList={pageList}
+            path={path}
+          />
+        </DowntimeNotification>
       </div>
     );
   }
 }
 
-function mapStateToProps(state, ownProps) {
-  const route = ownProps.route;
-  const { formConfig, pageList, path } = route;
-
-  return {
-    formConfig,
-    pageList,
-    path,
-    route,
-  };
-}
-
-const mapDispatchToProps = {};
```

Comment:
These were removed because the props were _only_ being set based off of other props. `state` isn't used at all, and we aren't passing any imported values in as props.